doc/articles/get-started-licensing.md

```diff
@@ -0,0 +1,63 @@
+
+## Create your account
+
+1. Go to our website, [platform.uno](https://platform.uno/), and click on the **Sign in** button in the top right corner, or go directly to [platform.uno/my-account](https://platform.uno/my-account).
+2. Enter your email address and click on **Register**.
+3. On the registration page, fill in your information, including your email, name, country, and password. Once done, click on **Sign up**.
+4. You will receive a confirmation email from **no-reply@platform.uno**. Click on **Confirm Email** to activate your account.
+5. You should then see the Sign in page. Enter your email and password and click on **Sign in** to access your account details, where you can update information or add more details.
+
+## Sign in to your IDE of choice
+
+After creating your Uno Platform account, follow the steps below to sign in to your preferred IDE:
+
+**I am developing on...**
+
+### [**Visual Studio 2022**](#tab/vs2022)
+
+If you’ve already set up **Visual Studio 2022** by following [Get Started on Visual Studio 2022](xref:Uno.GettingStarted.CreateAnApp.VS2022), you can sign in with your Uno Platform account as follows:
+
+1. Create a new Uno Platform project or open an existing one with `Uno.Sdk` version 5.5 or higher.
+2. After opening your project, a popup should appear. Click **Sign In / Register**.
+   ![Visual Studio 2022 Popup](Assets/uno-settings-popup.png)
+   >![NOTE]
+   > If the popup doesn’t appear, access Uno Platform settings by clicking on **View** > **Other Windows** > **Uno Platform Settings**.
+   > ![Visual Studio 2022 Menu](Assets/uno-settings-vs.png)
+
+### [**Visual Studio Code**](#tab/vscode)
+
+If you’ve already set up **Visual Studio Code** by following [Get Started on VS Code documentation](xref:Uno.GettingStarted.CreateAnApp.VSCode), sign in as follows:
+
+1. Create a new Uno Platform project or open an existing one with `Uno.Sdk` version 5.5 or higher.
+2. After opening your project, click the red Solution Button in the bottom status bar and switch to `.csproj` instead of `.sln`.
+   ![VS Code Solution Button](Assets/uno-vsc-csproj.gif)
+3. Access **Uno Platform** extension settings by selecting **View** > **Command Palette...** and typing `Uno Platform: Open Settings`.
+   ![Visual Studio Code Menu](Assets/uno-settings-vsc.png)
+
+### [**JetBrains Rider**](#tab/rider)
+
+If you’ve already set up **JetBrains Rider** by following [Get Started on JetBrains Rider documentation](xref:Uno.GetStarted.Rider), sign in as follows:
+
+1. Create a new Uno Platform project or open an existing one with `Uno.Sdk` version 5.5 or higher.
+2. Go to **Tools** > **Uno Platform** > **Settings...**.
```

Comment:
Missing steps no? We should have naggers for all 3 IDEs
cf. design:
![image](https://github.com/user-attachments/assets/76e04f6f-226e-40f4-905b-c91e3d322fa5)